CHANGELOG.md

```diff
@@ -14,6 +16,7 @@
 
 * Limit the number to parallel request to CDF. Should avoid getting rate limiting errors.
 * Add metrics for number of request going to CDF
+* Added `audience` option for Aize and AKSO OAuth2 flow. Removed default value for `scope` to support audience.
```

Comment:
fixed the enhancement that is shifted to the wrong version.

Reply:
Ahaaaaaaa, this is what created the confusion :sweat_smile: